components/HelpScreen.tsx

```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
         <ScrollView>
           <Column fill padding="10" align="space-between">
             <Text margin="7" style={Theme.TextStyles.header}>
-              {t('whatIsDigitalCredential?')}
+              {t('whatIsAnId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>{t('detail-10')}</Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
+              {t('whatAreTheDifferentTypesOfId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>
+              {t('detail-11')}
+              {hyperLinkString(
+                'here',
+                diffTypesOfIdsUrl,
+                '1.2.0/id-lifecycle-management/identifiers',
+              )}
+            </Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
```

Comment:
same here as well
https://github.com/mosip/inji/pull/1213/files#r1470853503

Reply:
No. We can't

Reply:
Then Lets add this style as separate style in themes and include that style in the array.

Reply:
Done.